pos_loyalty/models/loyalty_reward.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2004-2010 OpenERP SA
+# Copyright 2017 RGB Consulting S.L. (https://www.rgbconsulting.com)
+# Copyright 2018 Lambda IS DOOEL <https://www.lambda-is.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class LoyaltyReward(models.Model):
+    _name = "loyalty.reward"
+
+    name = fields.Char(string="Reward Name", size=32, index=True, required=True)
```

Comment:
I'm wondering why `size` and `index` are required here.
If there's no good reason, I'd drop them